R/theme.r

```diff
@@ -195,7 +196,11 @@
 #' p1 + theme(axis.line = element_line(size = 3, colour = "grey80"))
 #' p1 + theme(axis.text = element_text(colour = "blue"))
 #' p1 + theme(axis.ticks = element_line(size = 2))
-#' p1 + theme(axis.ticks.length = unit(.25, "cm"))
+#' p1 + theme(
+#'            axis.ticks.length.y = unit(.25, "cm"),
+#'            axis.ticks.length.x = unit(-.25, "cm"),
+#'            axis.text.x=element_text(margin=margin(t=.3, unit="cm"))
+#' )
```

Comment:
Please use tidyverse indenting here (2 spaces only) and make sure there are two spaces around each `=`. Also, please add a comment explaining what this example shows (inward/outward pointing ticks on x/y axes).